package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "eslint --ext .js",
     "server-test": "mocha -R spec --recursive",
     "browser-test": "karma start",
-    "test": "npm run server-test && npm run browser-test"
+    "test": "npm run server-test"
```

Comment:
Why?

Maybe because it's not convenient to run browser tests always. Okay. However, I don't see respective change, which would add `npm run browser-test` to CI config.

**BLOCKING**